develop/tutorials/articles/03-portlets/packaging-a-jsf-application.markdown

```diff
@@ -20,16 +20,12 @@ JSF framework expects a WAR layout and often requires the
 a JSF portlet so it can be treated as an OSGi module at runtime, and the pros
 and cons behind both.
 
-Liferay 7.0 supports the OSGi WAB standard for deployment of Java EE style WARs.
-Simply put, a WAB is an archive that that has a WAR layout and contains a
+Liferay 7.0 supports the OSGi Web Archive Bundle (WAB) standard for deployment of Java EE style WARs.  Simply put, a WAB is an archive that has a WAR layout and contains a
```

Comment:
Not according to Peter Kriens here:
http://www.aqute.biz/Bnd/Format#wab
But, Ok, I will change this.